docs/devguide/metricset-details.asciidoc

```diff
@@ -148,7 +156,7 @@ We recommend that you use all three when you create a metricset. Unit tests are
 written in Go and have no dependencies. Integration tests are also written
 in Go but require the service from which the module collects metrics to also be running.
 System tests for Metricbeat also require the service to be running in most cases and are
-written in Python based on our small Python test framework.
+written in Python 2 based on our small Python test framework.
```

Comment:
same comment...use an asciidoc attribute to resolve the version number.

Reply:
Fixed! :)